### PR TITLE
Add release note for 2.7.41491.1007

### PR DESCRIPTION
**Pull request by ZhidongPeng**
Add release note for 2.7.41491.1007

### CODE REVIEW

**Review by jacwil — CHANGES_REQUESTED** [review]
Per Aymendje, please upload release binaries to https://github.com/Azure/VMAgent/releases and not the source code.

**Review by Aymendje — CHANGES_REQUESTED** [review]
Please move any binary consumed by customers to a release instead.

File: release-notes/windows/2.7.41491.1007.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Release Notes for 2.7.41491.1007
+
+## Deployment Schedule
+
+Stage            | Date         | Regions
```
Comment (leisu001) [inline release-notes/windows/2.7.41491.1007.md:5]: should we add the AP data stages?

Comment (ZhidongPeng) [reply release-notes/windows/2.7.41491.1007.md]: AP data stages are too much for our customers.

**Review by jacwil — APPROVED** [review]

**Review by Aymendje — APPROVED** [review]
LGTM

**Pull request merged**